examples/test_examples.py

```diff
@@ -104,6 +104,10 @@ def test_textual_inversion(self):
                 --learnable_property object
                 --placeholder_token <cat-toy>
                 --initializer_token a
+                --validation_prompt <cat-toy>
+                --validation_steps 1
+                --save_steps 1
+                --num_vectors 2
```

Comment:
Why is this 2?

Reply:
Ah got it. cat toy.

Reply:
Actually it doesn't really matter, it could also be 3 or 4

Reply:
Could you shed a bit more details as to why?

Reply:
Essentially it just means "How many textual embedding vectors should be associated with the placeholder token" 

We can match 1 vector to `<cat-toy>` by creating the following token<>id matching:

<cat-toy>: vec_id_1

we can match two or more by doing:
<cat-toy>: vec_id_1
<cat-toy>_1: vec_id_2
....
<cat-toy>_n: vec_id_{n+1}

We can do this thanks to: https://github.com/huggingface/diffusers/blob/a121e05feb2e6e4ca02c7bc51fae7019e3005d18/src/diffusers/loaders.py#L391

Reply:
🧠 

Thanks!